R/description.R

```diff
@@ -176,7 +176,7 @@ use_description_field <- function(name, value, overwrite = FALSE) {
   }
 
   ui_done("Setting {ui_field(name)} field in DESCRIPTION to {ui_value(value)}")
-  desc$set(name, value)
+  desc$set_list(name, value)
```

Comment:
So this changes `use_description_field()` to always use the list form? :thinking:

In which case, have you taken a look at `use_description_list()`, which I see mentioned in a comment above, just to contemplate this pair of functions together?